kolibri/utils/options.py

```diff
@@ -32,6 +32,28 @@
             "envvars": ("KOLIBRI_DATABASE_HOST",),
         },
     },
+    "Server": {
+        "THREADS_POOL": {
```

Comment:
While the ENVVAR states that these are threads for cherrypy, the setting key itself doesnt?

Reply:
I didn't want to do the keys so long, and as far as they are under the `Server` Section, I considered it implicit.